Sources/Redis/Client/Commands/Redis+SortedSet.swift

```diff
@@ -0,0 +1,84 @@
+// MARK: Sorted Set commands
+extension RedisClient {
+    /// Adds all the specified members with the specified scores
+    /// to the sorted set stored at key
+    ///
+    /// https://redis.io/commands/zadd
+    public func zadd(_ key: String, items: [(String, RedisData)], options: [String]=[]) -> Future<Int> {
+        var args = [RedisData(bulk: key)] + options.map { RedisData(bulk: $0) }
+        for (score, member) in items {
+            args.append(RedisData(bulk: score))
+            args.append(member)
+        }
+
+        return command("ZADD", args).map(to: Int.self) { data in
+            guard let value = data.int else {
+                throw RedisError(identifier: "zadd", reason: "Could not convert resp to int.")
+            }
+            return value
+        }
+    }
+
+    /// Returns the number of elements in the sorted set at key with
+    /// a score between min and max.
+    ///
+    /// https://redis.io/commands/zcount
+    public func zcount(_ key: String, min: String, max: String) -> Future<Int> {
+        let args = [RedisData(bulk: key), RedisData(bulk: min), RedisData(bulk: max)]
+
+        return command("ZCOUNT", args).map(to: Int.self) { data in
+            guard let value = data.int else {
+                throw RedisError(identifier: "zcount", reason: "Could not convert resp to int.")
+            }
+            return value
+        }
+    }
+
+    /// Returns the specified range of elements in the sorted set stored
+    /// at key.
+    ///
+    /// https://redis.io/commands/zrange
+    public func zrange(_ key: String, start: Int, stop: Int, withScores: Bool=false) -> Future<[RedisData]> {
```

Comment:
`=` should have spaces on left and right (i.e., `" = "`)

Reply:
done